### ISSUE TITLE
Problems with setTemplate

### ISSUE THREAD

**Issue by funivan** [Respect/Validation#619]
This code does not change exception message

``` php
    Validator::instance(\stdClass::class)->setTemplate('invalid object')->check('test');
```

This works fine

``` php
(new Instance(\stdClass::class))->setTemplate('invalid object')->check('test');
```

How to set custom template when I use fluent interface of Validator object?

**Comment by henriquemoody** [comment]
Thanks for reporting, looks like it's a bug indeed.

That's because `setTemplate()` doesn't set the template for its children.

I will take a closer look to this bug as soon as possible.

**Comment by fracz** [comment]
Any progress on this?

**Comment by funivan** [comment]
Any news?

**Comment by henriquemoody** [comment]
It should be fixed in the version 1.1.19 which I intent to release very soon.

Thank you very much for reporting! 🐼
